packages/dev/core/src/Engines/Extensions/engine.views.ts

```diff
@@ -157,7 +157,7 @@ Engine.prototype.unRegisterView = function (canvas: HTMLCanvasElement): Engine {
 };
 
 Engine.prototype._renderViews = function () {
-    if (!this.views) {
+    if (!this.views || this.views.length === 0) {

```

Comment:
You should add `this.activeView = null;` before returning `false`, else it is a breaking change.